WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert Node Progress View

## Changes

- Converted Node Progress View
- Fixed score updating immediately in the workgroup row display

## Test

1. Open the Classroom Monitor for a run
2. Go to the "Grade By Step" view
3. Expand a lesson
4. Click on a step. Make sure everything works properly like the student work and scores show up.
5. Change the score for a workgroup multiple times and make sure it updates immediately each time in the workgroup row.
6. Change the max score and make sure it updates immediately everywhere.
7. Go to another step using the drop down or arrow buttons.
8. Make sure everything is updated with the correct values for status and score and student work for the new step.

Closes #880
